README.md

```diff
@@ -60,3 +60,25 @@ parameters:
                 - 2     # /Home
                 - 43    # /Media
 ```
+
+## Default location for content type
+If suggested locations are given for content type (or default) the first location in the list will be preselected as location.
```

Comment:
If suggested locations are provided for a Content Type (or default) the first location in the list will be preselected.